### PR TITLE
Update OPA version with other dependecies 

### PR DESCRIPTION
**Pull request by anastasia-malysheva**
## How Has This Been Tested?

- [ ] Added unit testing to cover
- [ ] Tested manually
- [ ] Tested by integration testing
- [ ] Have not tested

## Types of changes

- [ ] Bug fix
- [ ] New functionallity
- [ ] Documentation
- [ ] Refactoring
- [ ] CI

### CODE REVIEW

**Comment by denis-tingaikin** [comment]
@fkautz 

Here I see two options on how to merge this:
1. Split this PR to small PR and update each required dependency for OPA separately.
2. Just merge this PR
3. Your option.


Thoughts?

**Comment by denis-tingaikin** [comment]
@anastasia-malysheva Have you updated all deps or only that related to opa?

**Comment by anastasia-malysheva** [comment]
> @anastasia-malysheva Have you updated all deps or only that related to opa?

I've updated all deps.

**Comment by anastasia-malysheva** [comment]
@denis-tingaikin I've removed updates for dependencies which are not related to OPA.

File: pkg/networkservice/chains/endpoint/combine_test.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2021 Doc.ai and/or its affiliates.
+// Copyright (c) 2022 Doc.ai and/or its affiliates.
```
Comment (NikitaSkrynnik) [inline pkg/networkservice/chains/endpoint/combine_test.go:1]:
```suggestion
// Copyright (c) 2021-2022 Doc.ai and/or its affiliates.
```

File: pkg/networkservice/common/monitor/passthrough_test.go [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2021 Cisco and/or its affiliates.
+// Copyright (c) 2022 Cisco and/or its affiliates.
```
Comment (NikitaSkrynnik) [inline pkg/networkservice/common/monitor/passthrough_test.go:1]:
```suggestion
// Copyright (c) 2021-2022 Cisco and/or its affiliates.
```

**Review by denis-tingaikin — COMMENTED** [review]
@anastasia-malysheva Please resolve comments and this may be merged

File: pkg/networkservice/common/monitor/passthrough_test.go (hunk 2 above)
Comment (anastasia-malysheva) [reply pkg/networkservice/common/monitor/passthrough_test.go]: @NikitaSkrynnik fixed

**Review by denis-tingaikin — COMMENTED** [review]
@edwarnicke We're merging this to check NSM on CI with new deps.

We'll revert these changes if we'll get any regressions

**Review by denis-tingaikin — APPROVED** [review]

**Pull request merged**